teamcity-rest-client-api/src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -492,6 +524,35 @@ interface VcsRoot {
     val defaultBranch: String?
 }
 
+interface BuildAgent {
+    val id: BuildAgentId
+    val name: String
+    val pool: BuildAgentPool
+
+    val connected: Boolean
+    val enabled: Boolean
+    val authorized: Boolean
+    val uptodate: Boolean
+
+    val ip: String
```

Comment:
`ip` in lower-case may be confusing, I suggest to rename it to something like `ipAddress`.